systemd/step-ssh-cert-renewer@.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
```

Comment:
This is set up as a systemd template, which makes sense for `cert-renewer@.*` because there can be many X.509 certs on a systme. But, I think we should rename these SSH renewal timers to `cert-renewer-ssh.*` because there's typically only one SSH server on a machine.

Reply:
yes but it would need to run for 3 certificate types.

Reply:
In practice, you really only need an ECDSA or Ed25519 cert.
ECDSA has been supported since OpenSSH 5.7 (2011) and Ed25519 since OpenSSH 6.5 (2014).
You just don't get much in terms of broader compatibility by having more cert types, and it adds to your host management burden.
So, I suggest that it be a single renewer rather than a template, and people can always templatize it by renaming it if they need to.

None of this is in our documentation though, and people would probably benefit from docs around this renewer. I'll make an issue for that in [smallstep/docs](https://github.com/smallstep/docs/)